src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[fromFormat.length - 1]);
+  const result = [];
+
+  result[toFormat.indexOf('DD')] = dateArr[fromFormat.indexOf('DD')];
+  result[toFormat.indexOf('MM')] = dateArr[fromFormat.indexOf('MM')];
```

Comment:
Check that `fromFormat` and `toFormat` actually contain the appropriate date parts. 
This will ensure that errors are avoided if one of the formats is missing.
I used the renamed variable ``result``
 Example:
```suggestion

  if (toFormat.includes('DD')) {
    formattedDate[toFormat.indexOf('DD')] = dateArr[fromFormat.indexOf('DD')];
  }
  if (toFormat.includes('MM')) {
    formattedDate[toFormat.indexOf('MM')] = dateArr[fromFormat.indexOf('MM')];
  }
```